examples/src/main/java/com/mapbox/navigation/examples/ui/BuildingFootprintHighlightActivityKt.kt

```diff
@@ -47,7 +48,11 @@ class BuildingFootprintHighlightActivityKt : AppCompatActivity(), OnNavigationRe
         setContentView(R.layout.activity_final_destination_arrival_building_highlight)
 
         navigationView.onCreate(savedInstanceState)
-        navigationView.initialize(this, getInitialCameraPosition())
+        navigationView.initialize(
+            this,
+            getInitialCameraPosition(),
+            Utils.getMapboxAccessToken(this)
```

Comment:
did we really need to introduce this access token requirement here? it's required in a few places and it breaks APIs to change it

cc: @abhishek1508 @Guardiola31337 @LukasPaczos